pkg/loki/modules.go

```diff
@@ -149,7 +149,7 @@ func (t *Loki) initRuntimeConfig() (services.Service, error) {
 	validation.SetDefaultLimitsForYAMLUnmarshalling(t.Cfg.LimitsConfig)
 
 	var err error
-	t.runtimeConfig, err = runtimeconfig.NewRuntimeConfigManager(t.Cfg.RuntimeConfig, prometheus.DefaultRegisterer)
+	t.runtimeConfig, err = runtimeconfig.New(t.Cfg.RuntimeConfig, prometheus.DefaultRegisterer, util_log.Logger)
```

Comment:
Note that dskit's runtimeconfig has metrics without "cortex_" prefix. If prefix is not used, metrics rename is worth mentioning somewhere. Loki project may also prefer to use "loki_" prefix instead.

Reply:
@pstibrany oh yeah, good catch. Thanks! @slim-bean WDYT?

Reply:
+1 for having ability to choose prefix for runtime metrics!. 

btw, Q: Looks like previously it was named with prefix `cortex_` and now without prefix.

Won't that break some stuff in `deployment_tools`?

```
ksonnet/vendor/github.com/grafana/cortex-jsonnet/cortex-mixin/alerts/alerts.libsonnet
124:            cortex_runtime_config_last_reload_successful == 0

ksonnet/lib/tempo-cloud-mixin/alerts.libsonnet
36:                cortex_runtime_config_last_reload_successful{namespace=~"%s"} == 0
```

Reply:
@kavirajk I can add back the `cortex_` prefix. Omitted it by mistake 🤦‍♂️ If we have to keep the `cortex_` prefix, I can add it back here (in the PR).

Does the deployment_tools snippet you pasted relate also to Loki? I.e., Loki has to keep the `cortex_` prefix on metrics?

Reply:
Thanks @aknuds1 :)  I quickly checked in deployment tools repo and looks like Loki doesn't use this metric in any dashboards or alerts. The ones I pasted are from Cortex and Tempo.

I see two options:
1. keep `cortex_` prefix as default and have ability to change the prefix when needed.
2. No default, just accept prefix as argument.

In case 2, we should careful when Tempo or Cortex uses this change in their releases. (they should update the alerts accordingly)

Both options are fine IMO.

Reply:
I will fix the PR to add back the `cortex_` prefix.

Reply:
Added back `cortex_` prefix.